### PR TITLE
Add evm query

### PR DESCRIPTION
**Pull request by philipsu522**

### CODE REVIEW

File: packages/sei-cosmwasm/README.md [hunk 1]
```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
             - Change the Admin of the Denom. Only the current admin can change the admin.
         - SetMetadata
             - Set the denom metadata of a factory denom. Only the current admin can set metadata.
+- EVM
+    - Query
+        - StaticCall
+            - TODO
```
Comment (philipsu522) [inline packages/sei-cosmwasm/README.md:63]: not sure what this does

File: packages/sei-cosmwasm/README.md [hunk 2]
```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
...
+        - Erc20TransferPayload
```
Comment (philipsu522) [inline packages/sei-cosmwasm/README.md:64]: want to confirm these are all "Gets"

File: packages/sei-cosmwasm/README.md [hunk 3]
```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
...
+        - Erc721TransferPayload
+            - Similar to the Erc20 equivalent
+        - Erc721ApprovePayload
+            - Similar to the Erc20 equivalent
+        - Erc721Approved
+            - Check si a caller is approved based on caller, contract address and token id
+        - Erc721IsApprovedForAll
+            - Checks if the caller is approved for all based on caller, contract address, owner and operator
+        - Erc721SetApprovalAllPayload
+            - Gets the Erc721 SetApproveAll payload from caller, contract address, owner and operator
```
Comment (philipsu522) [inline packages/sei-cosmwasm/README.md:85]: is this correct? seems like it's "setting" something

File: packages/sei-cosmwasm/README.md [hunk 4]
```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
...
+        - Erc20Balance
+            - Gets the Erc20 balance from contract address and account
+        - Erc721TransferPayload
+            - Similar to the Erc20 equivalent
+        - Erc721ApprovePayload
+            - Similar to the Erc20 equivalent
+        - Erc721Approved
+            - Check si a caller is approved based on caller, contract address and token id
+        - Erc721IsApprovedForAll
+            - Checks if the caller is approved for all based on caller, contract address, owner and operator
```
Comment (philipsu522) [inline packages/sei-cosmwasm/README.md:83]: - is this actually verifying the caller is approved or the operator
- what specific operations does "approved for all" mean? i can add more info here

File: packages/sei-cosmwasm/README.md [hunk 5]
```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
...
+        - Erc20TokenInfo
+            - Gets the Erc20 token info from contract address and caller
+        - Erc20Balance
+            - Gets the Erc20 balance from contract address and account
+        - Erc721TransferPayload
+            - Similar to the Erc20 equivalent
+        - Erc721ApprovePayload
+            - Similar to the Erc20 equivalent
+        - Erc721Approved
+            - Check si a caller is approved based on caller, contract address and token id
```
Comment (philipsu522) [inline packages/sei-cosmwasm/README.md:81]: what operations is the caller approved for here?

File: packages/sei-cosmwasm/README.md [hunk 6]
```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
...
+- EVM
+    - Query
+        - StaticCall
+            - TODO
+        - Erc20TransferPayload
+            - Gets the Erc20 transfer payload from on recipient and amount
+        - Erc20TransferFromPayload
+            - Gets the Erc20 transfer from payload based on owner, recipient and amount
+        - Erc20ApprovePayload
+            - Gets the Erc20 approve payload from spender and amount
```
Comment (philipsu522) [inline packages/sei-cosmwasm/README.md:69]: verify this is correct

File: packages/sei-cosmwasm/README.md [hunk 7]
```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
...
         - SetMetadata
             - Set the denom metadata of a factory denom. Only the current admin can set metadata.
+- EVM
+    - Query
+        - StaticCall
+            - TODO
+        - Erc20TransferPayload
+            - Gets the Erc20 transfer payload from on recipient and amount
+        - Erc20TransferFromPayload
+            - Gets the Erc20 transfer from payload based on owner, recipient and amount
```
Comment (philipsu522) [inline packages/sei-cosmwasm/README.md:67]: verify this is correct

File: packages/sei-cosmwasm/README.md [hunk 8]
```diff
@@ -57,6 +57,42 @@ Currently, Sei Bindings support query and message support for the sei custom mod
...
+        - Erc20TokenInfo
+            - Gets the Erc20 token info from contract address and caller
+        - Erc20Balance
+            - Gets the Erc20 balance from contract address and account
+        - Erc721TransferPayload
+            - Similar to the Erc20 equivalent
+        - Erc721ApprovePayload
+            - Similar to the Erc20 equivalent
+        - Erc721Approved
+            - Checks if a caller is approved based on caller, contract address and token id
```
Comment (philipsu522) [inline packages/sei-cosmwasm/README.md:81]: what operations is the caller approved for here?

File: packages/sei-cosmwasm/README.md (hunk 1 above)
Comment (codchen) [reply packages/sei-cosmwasm/README.md:63]: this is basically the "generic query" endpoint (i.e. query any endpoint on any EVM contract)

File: packages/sei-cosmwasm/README.md (hunk 2 above)
Comment (codchen) [reply packages/sei-cosmwasm/README.md:64]: yeah these are just helpers for callers to format payload. Alternatively they can also format payload by themselves by reading ERC20's ABI in rust

File: packages/sei-cosmwasm/README.md (hunk 7 above)
Comment (codchen) [reply packages/sei-cosmwasm/README.md:67]: yes

File: packages/sei-cosmwasm/README.md (hunk 6 above)
Comment (codchen) [reply packages/sei-cosmwasm/README.md:69]: yes

File: packages/sei-cosmwasm/README.md (hunk 8 above)
Comment (codchen) [reply packages/sei-cosmwasm/README.md:81]: to send NFTs on behalf of the owner

File: packages/sei-cosmwasm/README.md (hunk 4 above)
Comment (codchen) [reply packages/sei-cosmwasm/README.md:83]: this checks if the operator is approved to operate (aka transfer on behalf of the owner) all NFTs under the owner

File: packages/sei-cosmwasm/README.md (hunk 3 above)
Comment (codchen) [reply packages/sei-cosmwasm/README.md:85]: yeah this is correct (there is a function `SetApproveAll` that approves the operator to operate all tokens one owns)

**Pull request merged**